templates/star2lte_template.md

```diff
@@ -66,7 +66,7 @@ Please download the belowed needed files and tools:
     - The first boot may take longer, and at least one spontaneous reboot is expected during the process
     - You should be greeted with the lock screen, the default password is `1234`
 
-Congratulations, if everything went well, now you should be running Droidian.
```

Comment:
These changes should be sufficient , the changes will reflect in files generated inside content/devices folder.